lighthouse-core/test/config/config-test.js

```diff
@@ -721,6 +721,19 @@ describe('Config', () => {
       assert.strictEqual(config.categories['lighthouse-plugin-simple'].title, 'Simple');
     });
 
+    it('should append a group', () => {
```

Comment:
should also have one of the audits in the fixture using the group? (maybe better to check up in 'should append audits'?)

Reply:
Moved to 'should append audits' and added auditRef check.